malariagen_data/anopheles.py

```diff
@@ -0,0 +1,2213 @@
+import sys
+import warnings
+from abc import ABC, abstractmethod
+from collections import Counter
+from pathlib import Path
+from textwrap import dedent
+
+import allel
+import dask.array as da
+import ipinfo
+import numba
+import numpy as np
+import pandas as pd
+import xarray as xr
+import zarr
+from tqdm.auto import tqdm
+from tqdm.dask import TqdmCallback
+
+try:
+    # noinspection PyPackageRequirements
+    from google import colab
+except ImportError:
+    colab = None
+
+from . import veff
+from .util import (
+    DIM_ALLELE,
+    DIM_PLOIDY,
+    DIM_SAMPLE,
+    DIM_VARIANT,
+    LoggingHelper,
+    Region,
+    da_compress,
+    da_from_zarr,
+    dask_compress_dataset,
+    init_filesystem,
+    init_zarr_store,
+    locate_region,
+    read_gff3,
+    resolve_region,
+    unpack_gff3_attributes,
+    xarray_concat,
+)
+
+DEFAULT_SITE_FILTERS_ANALYSIS = "dt_20200416"
+DEFAULT_GENOME_PLOT_WIDTH = 800  # width in px for bokeh genome plots
+DEFAULT_GENES_TRACK_HEIGHT = 120  # height in px for bokeh genes track plots
+
+
+class Anopheles(ABC):
```

Comment:
Suggest rename to "AnophelesDataResource" for naming consistency with plasmodium modules.